src/keys.rs

```diff
@@ -7,9 +7,17 @@
 use glib::translate::*;
 use ffi;
 use libc::c_uint;
+use std::char;
 
 pub fn keyval_name(keyval: u32) -> Option<String> {
     unsafe {
         from_glib_none(ffi::gdk_keyval_name(keyval as c_uint))
     }
 }
+
+pub fn keyval_to_unicode(keyval: u32) -> Option<char> {
+	unsafe {
+		let c = char::from_u32(ffi::gdk_keyval_to_unicode(keyval));
```

Comment:
We use spaces tabulation (' ') not tab tabulations ('\t'). On top of that, your indentation isn't good. Not good ! =D